webapp/graphite/render/glyph.py

```diff
@@ -899,16 +899,16 @@ def onHeaderPath(match):
           name = ''
           for char in re.findall(r'L -(\d+) -\d+', match.group(1)):
```

Comment:
might this also need to be a binary string, because match groups are bytes now?

Reply:
... also, `name` is a string, two lines above:

```python
          name = ''
```

 which you can't substitute into a bytes below:

```python
          return b'</g><g data-header="true" class="%s">' % name
```

example:

```
>>> name = ''
>>> b"foobar %s" % name
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: %b requires a bytes-like object, or an object that implements __bytes__, not 'str'
```

Reply:
(so you see why keeping all this string-composing logic in terms of strings rather than bytes might be easier to keep working and readable)